src/Control/Arrow/Kleisli.purs

```diff
@@ -14,6 +16,12 @@ instance semigroupoidKleisli :: (Monad m) => Semigroupoid (Kleisli m) where
 instance categoryKleisli :: (Monad m) => Category (Kleisli m) where
   id = Kleisli return
 
+instance profuncorKleisli :: (Monad m) => Profunctor (Kleisli m) where
```

Comment:
I think this can be relaxed to just `Functor` yes? Or do the laws require `Monad`?